examples/user_guide/Working_with_Streaming_Data.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
That's quite a long title; not sure if "Working with" adds anything? Maybe "Streaming Data"?

Reply:
Sure.

Reply:
Or maybe "Streaming Data Sources", if that helps avoid confusion with hv streams in general.